{"timestamp":"2025-10-29T10:53:34.482306","level":"info","event":"DAG bundles loaded: dags-folder","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-10-29T10:53:34.484164","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/Airflow.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-10-29T10:53:34.548139","level":"info","event":"Tmp dir root location: /tmp","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:34.549711","level":"info","event":"Running command: ['/usr/bin/bash', '-c', 'python3 /opt/airflow/scripts/ARPEGE.py']","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:34.568324","level":"info","event":"Output:","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.977814","level":"info","event":"docker: permission denied while trying to connect to the Docker daemon socket at unix:///var/run/docker.sock: Post \"http://%2Fvar%2Frun%2Fdocker.sock/v1.24/containers/create\": dial unix /var/run/docker.sock: connect: permission denied.","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.978217","level":"info","event":"See 'docker run --help'.","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.983517","level":"info","event":"üîÑ Descargando datos de http://127.0.0.1:8081/v1/forecast ...","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.984064","level":"info","event":"‚¨áÔ∏è Descargando temperature_2m del Bucket S3 de AWS...","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.984487","level":"info","event":"Traceback (most recent call last):","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.984982","level":"info","event":"  File \"/opt/airflow/scripts/ARPEGE.py\", line 177, in <module>","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.985244","level":"info","event":"    main()","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.985560","level":"info","event":"  File \"/opt/airflow/scripts/ARPEGE.py\", line 153, in main","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.985806","level":"info","event":"    df, download_time = fetch_forecast(lats, lons)","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.986137","level":"info","event":"                        ^^^^^^^^^^^^^^^^^^^^^^^^^^","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.986416","level":"info","event":"  File \"/opt/airflow/scripts/ARPEGE.py\", line 66, in fetch_forecast","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.986777","level":"info","event":"    subprocess.run([","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.987165","level":"info","event":"  File \"/usr/local/lib/python3.12/subprocess.py\", line 571, in run","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.987572","level":"info","event":"    raise CalledProcessError(retcode, process.args,","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:37.987963","level":"info","event":"subprocess.CalledProcessError: Command '['docker', 'run', '--rm', '-v', 'open-meteo-data:/app/data', 'ghcr.io/open-meteo/open-meteo', 'sync', 'meteofrance_arpege_europe', 'temperature_2m']' returned non-zero exit status 126.","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:38.178412","level":"info","event":"Command exited with return code 1","logger":"airflow.task.hooks.airflow.providers.standard.hooks.subprocess.SubprocessHook"}
{"timestamp":"2025-10-29T10:53:38.179650","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"AirflowException","exc_value":"Bash command failed. The command returned a non-zero exit code 1.","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":920,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1215,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/bash.py","lineno":232,"name":"execute"}],"is_group":false,"exceptions":[]}]}
